VAMobile/documentation/docs/Engineering/BackEnd/Architecture/FeatureFlagging.md

```diff
@@ -20,7 +20,8 @@ You will then need to add your production id.me email address to `config/setting
 ## Toggling Feature Flags
 
 Flipper URLS (note the 'api' in the url)
-* staging: https://staging-api.va.gov/flipper/features
-* production: https://api.va.gov/flipper/features
+
+* staging: staging-api.va.gov/flipper/features
+* production: api.va.gov/flipper/features
```

Comment:
removing `https://` so they are not represented as links in the doc and rather URLs the should be read out

Reply:
@brea11y would like your take on this issue